### PR TITLE
refactor: tracee-rules to use Event type from tracee-ebpf

### PR DESCRIPTION
**Pull request by itaysk**
close: #496, #494

### CODE REVIEW

File: tracee-rules/engine/engine.go [hunk 1]
```diff
@@ -5,6 +5,7 @@ import (
 	"log"
 
 	"github.com/aquasecurity/tracee/tracee-rules/types"
+	tracee "github.com/aquasecurity/tracee/tracee/external"
```
Comment (grantseltzer) [inline tracee-rules/engine/engine.go:8]: I'm confused by this import path. To me it looks like the `tracee/external` package is in `github.com/aquasecurity/tracee/tracee-ebpf/tracee/external`, how does this work when the `tracee-ebpf` part is dropped?

**Comment by grantseltzer** [comment]
Looks like you have failed tests, on my end #494 is resolved by this though. Otherwise code looks good to me.

File: tracee-rules/engine/engine.go (hunk 1 above)
Comment (itaysk) [reply tracee-rules/engine/engine.go:8]: you are looking at HEAD but go import gets the packages from latest tag. in tandem with the release we would need to updated these paths, and I've opened #497 as a reminder

**Comment by itaysk** [comment]
alright I've rebased on main and tests are now passing. we still have the caveat I mentioned in https://github.com/aquasecurity/tracee/pull/498#discussion_r569363976. if you have another idea to approach this, I'm open to suggestions

**Comment by grantseltzer** [comment]
Ah I follow, this LGTM then!

**Review by grantseltzer — APPROVED** [review]

**Pull request merged**